tests/node-app.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: node-http-deployment
+  labels:
+    app: node-http
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: node-http
+  template:
+    metadata:
+      labels:
+        app: node-http
+    spec:
+      containers:
+        - name: node-http
+          image: ghcr.io/metalbear-co/mirrord-node:latest
+          ports:
+            - containerPort: 80
```

Comment:
health checks

Reply:
Yeah I'm still working on the tests, and I didn't push most of what I already have for tests. I created a deployment with liveness probes in the `test-images` repo, and used it to test manually, but didn't push it yet because I don't yet know if I'll use it in the tests.